### DISCUSSION TITLE
Unable to add image using the Tip Tap Rich Text Editor

### DISCUSSION THREAD

**Discussion by Jimam-Tamimi** [mantinedev/mantine#5507] [Feature requests]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.3.2

### What package has an issue?

@mantine/tiptap

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

All

### Describe the bug

In the official [docs ](https://mantine.dev/others/tiptap/) of the Rich Text Editor I couldn't find any details related to adding image using the Rich Text Editor 

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
You can use any TipTap extensions with RichTextEditor component – https://tiptap.dev/extensions
